gengokit/httptransport/templates_test.go

```diff
@@ -173,12 +175,9 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 		fmt.Printf("Error while reading path params: %v\n", err)
 		return nil, errors.Wrap(err, "couldn't unmarshal path parameters")
 	}
-	queryParams, err := QueryParams(r.URL.Query())
+
+	queryParams := r.URL.Query()
```

Comment:
Pretty necessary, that line is part of a larger template for the DecodeHTTPEndpointRequest.

The template for the individual queryparams relies on `queryParams` existing. Otherwise I'd need to use r.URL.Query() for every queryparam